ChangeLog

```diff
@@ -314,7 +314,7 @@
   * [Rework] Rework tags parsing machine
   * [Rework] Slightly improve old regexp API
   * [Rework] Start conversion of the redis pool code to c++
-  * [Rework] Try to resolve failed upstreams more agressively
+  * [Rework] Try to resolve failed upstreams more aggressively
```

Comment:
Some projects don't like changing the change log, let me know, it's trivial for me to drop a file or files...

Reply:
Indeed, the ChangeLog is autogenerated (semi) from a git messages, so I would leave it as is.